build.gradle

```diff
@@ -61,12 +61,11 @@ dependencies {
     testImplementation (group: 'io.github.bonigarcia', name: 'webdrivermanager', version: '5.3.3') {
         exclude group: 'org.seleniumhq.selenium'
     }
-    testImplementation ('org.seleniumhq.selenium:selenium-chrome-driver') {
-        version {
-            strictly "[${seleniumVersion}, 5.0)"
-            prefer "${seleniumVersion}"
-        }
-    }
+    testImplementation platform(group: 'org.seleniumhq.selenium', name: 'selenium-bom', version: '4.11.0')
```

Comment:
are you sure you want to merge this? how about `seleniumVersion` then?

Reply:
`seleniumVersion ` will be used in published POM and will declare minimal support version of Selenium
as for the test stage Gradle will resolve conflicts and will pick up the latest version (== declared in test scope), so all tests will be run against the latest version

Reply:
if this is only for tests then should be ok